### DISCUSSION TITLE
Arch's AUR is the "wrong" place

### DISCUSSION THREAD

**Discussion by ojaksch** [MiSTerArch/PKGBUILDs#10] [General]
I'm using ArchLinux since 2011, coming from Gentoo. So nothing new to me and I welcome MiSTerArch!
But from my own experience in maintaining some dozen of AUR packages for a few years, I recommend you to NOT use Arch's AUR system for arm-only packages. I'd some similar packages that got deleted during a "non x86-64 cleanup" last year, without announcing that and no replies to my concerns, so I dropped _all_ my packages.
Better think about an self-hosted MiSTerArch one has to include to pacman.conf

**Comment by amstan** [comment]
Good to know. I guess this would happen even for .any packages? I'm thinking about the .rbf cores, they're not exactly arm binaries.

**Comment by ojaksch** [comment]
I'm sure that this cleanup relates to all arm-only packages. Packages available for x86_64 _and_ arm* were okay, but my armv7 and aarch64 packages got deleted.

**Comment by amstan** [comment]
Yeah, I'm still thinking about this. Would like to reasearch it more. I would still like perhaps some people to be able to publish mister files directly to the AUR, otherwise this whole thing has to stay super centralized due to how there's only one misterarch.hypertriangle.com repo.

**Comment by amstan** [comment]
So I'm working on mister-downloader, #20. That should totally be on the aur. I'm using it on my desktop and it's awesome.
I've also been thinking about packages with cores in them, #21, also something I would want to have on my desktop in case I'm doing sshfs shares.

So all of those packages can be `arch=('any')`.

**Reply by ojaksch** [reply]
> So all of those packages can be `arch=('any')`.

_any_ is okay as for many (all?) perl packages, for example. Hope they don't change their attitude as your packages are _for_ a non x86* system...
